backend/query/src/models/edit-suggestion.model.ts

```diff
@@ -0,0 +1,84 @@
+import { AggregateRoot } from '@nestjs/cqrs';
+import { prop } from '@typegoose/typegoose';
+import { Expose, Type } from 'class-transformer';
+import { IsArray, IsInt, IsOptional, IsString } from 'class-validator';
+import { SUGGESTION_STATUS } from '../shared/enums/suggestion-status.enum';
+
+export class SuggestionTag {
+  @prop({ type: [String] })
+  @IsArray()
+  @IsOptional()
+  @IsString({ each: true })
+  push?: string[];
+
+  @prop({ type: [String] })
+  @IsArray()
+  @IsOptional()
+  @IsString({ each: true })
+  pull?: string[];
```

Comment:
Das hatten wir ja jetzt schon aber pull als bezeichner zu nehmen bei einem API Design welches sich an DDD orientiert wiederspricht sich. Ich wäre für: `add`|`delete`, `insert`:`remove`. Ist das eine Option für dich?

Reply:
Nicht teil dieses Issues

Reply:
Sowie deine ganzen Formatting sachen in diesem PR? xD

Reply:
push/pull sind in diesem Fall genauso valide wie jede andere Option. Ich stimme @Branchverse allerdings zu: Nicht teil dieser issue bzw. von diesem PR